devices/Sht4x/Sht4X.cs

```diff
@@ -0,0 +1,136 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Sht4x
+{
+    /// <summary>
+    /// Humidity and Temperature Sensor Sht4X.
+    /// </summary>
+    public class Sht4X : IDisposable
+    {
+        /// <summary>
+        /// Default I2C address of the Sht4x sensor.
+        /// </summary>
+        public const byte DefaultAddress = 0x44;
```

Comment:
I think we're using `DefaultI2cAddress` overall

Reply:
fixed